static/templates/stream_member_list_entry.hbs

```diff
@@ -1,6 +1,8 @@
 <tr data-subscriber-email="{{email}}">
     <td class="subscriber-name">{{name}}</td>
+    {{#if show_email}}
     <td class="subscriber-email">{{email}}</td>
+    {{/if}}
```

Comment:
It turns out this change breaks the "unsubscribe" buttons, because they were using this element.  I fixed that in the d0927d38fc4f3219cd350f79db6b0f3e3bfc8874 prep commit.

In the future, please be more careful in your QA to test UI widgets that might be affected by changes and search the codebase for potential issues caused by a change (`git grep subscriber-email` would have found this bug).

Reply:
Yeah I will take care of this in future.